float64 time
float64[700] q_pred
float64[700] dotq_pred
float64[700] ddotq_pred
float64[700] tau_pred